src/test/java/seedu/address/model/concert/ConcertContactTest.java

```diff
@@ -3,29 +3,77 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalConcertContacts.ALICE_COACHELLA;
+import static seedu.address.testutil.TypicalConcertContacts.ALICE_GLASTONBURY;
+import static seedu.address.testutil.TypicalConcertContacts.BOB_COACHELLA;
 import static seedu.address.testutil.TypicalConcerts.COACHELLA;
+import static seedu.address.testutil.TypicalConcerts.GLASTONBURY;
 import static seedu.address.testutil.TypicalPersons.ALICE;
 import static seedu.address.testutil.TypicalPersons.BOB;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.person.Person;
+
 public class ConcertContactTest {
     @Test
     void createConcertContact() {
-        //checks if ConcertContact object can be created
+        // checks if ConcertContact object can be created
         ConcertContact contactTest1 = new ConcertContact(ALICE, COACHELLA);
 
         assertEquals(COACHELLA, contactTest1.getConcert());
         assertEquals(ALICE, contactTest1.getPerson());
     }
 
     @Test
-    void equalsTest() {
+    public void isSameConcertContact() {
+        // same concert returns true
+        assertTrue(ALICE_COACHELLA.isSameConcertContact(ALICE_COACHELLA));
+
+        // different person returns false
+        assertFalse(ALICE_COACHELLA.isSameConcertContact(BOB_COACHELLA));
+
+        // different concert returns false
+        assertFalse(ALICE_COACHELLA.isSameConcertContact(ALICE_GLASTONBURY));
+    }
+
+    @Test
+    public void isAssociated_person() {
+        // person in concertContact returns true
+        assertTrue(ALICE_COACHELLA.isAssociated(ALICE));
+
+        // person not in concertContact returns false
+        assertFalse(ALICE_COACHELLA.isAssociated(BOB));
+
+        // person is null returns false
+        assertFalse(ALICE_COACHELLA.isAssociated((Person) null));
+    }
+
+    @Test
+    public void isAssociated_concert() {
+        // concert is the same returns true
+        assertTrue(ALICE_COACHELLA.isAssociated(COACHELLA));
+
+        // concert different returns false
+        assertFalse(ALICE_COACHELLA.isAssociated(GLASTONBURY));
+
+        // null returns false
+        assertFalse(ALICE_COACHELLA.isAssociated((Concert) null));
+    }
+    @Test
+    void equals() {
```

Comment:
include public access modifier to follow the rest. Also, add a line before @Test annotation in Line 63